### ISSUE TITLE
upload library to maven repo

### ISSUE THREAD

**Issue by dbachelder** [askerov/DynamicGrid#21]
so we can include as normal gradle dependency

**Comment by denisk20** [comment]
@askerov First, good to see that the project keeps going. Second, I can create a pull request with necessary gradle code for uploading to maven central, if you're interested.

**Comment by askerov** [comment]
@denisk20 I think today this library already not really actual. It's time to implements such things with RecyclerView and drag/drop framework. As well I have no time to support it reasonably well, so I don't want to upload it to maven central.

**Comment by denisk20** [comment]
@askerov Sorry to hear that, but I think that your library is much easier to use than drag and drop framework. I'm sure many people think so too (based on this repo's activity). Putting the library into maven central doesn't force you to keep any schedule for updates, but will make it much easier for people to use the library. This increases chances that someone might be willing to contribute.

**Comment by fhur** [comment]
I agree with @denisk20 also why not consider giving @denisk20 (which is very active in DynamicGrid) the push/merge rights?

**Comment by denisk20** [comment]
I'd be fine with that, the library helped me a lot and I'd be glad to give back.

**Comment by michael-quinlan** [comment]
@denisk20 did you end up publishing this?

**Comment by denisk20** [comment]
@michael-quinlan No, I haven't heard back from @askerov.
